toolbox/db/db_get.m

```diff
@@ -219,6 +219,48 @@
             varargout{1} = sAnatFiles;
         end
 
+%% ==== ANATOMY FILE ====
+    % Usage: [sFiles, sAnatomy] = db_get('AnatomyFile', FileIDs)
+    % Usage: [sFiles, sAnatomy] = db_get('AnatomyFile', FileNames)
+    % Usage: [sFiles, sAnatomy] = db_get('AnatomyFile', CondQuery)
+    case 'AnatomyFile'
```

Comment:
A 'fields' argument would be worth it to optimize some queries that don't require all columns.

Reply:
Added in c1b8e36.
It also adds the parsing to Anatomy or Surface db_template structures